Cargo.toml

```diff
@@ -13,9 +13,19 @@ Simple numerics. This crate contains basic arbitrary-sized integer,
 rational, and complex types.
 """
 
-[dependencies]
-rustc-serialize = "0.3.13"
-rand = "0.3.8"
+[dependencies.rustc-serialize]
+version = "0.3.13"
+optional = true
+
+[dependencies.rand]
+version = "0.3.8"
+optional = true
+
+[features]
+
+rational = []
+bigint = ["rustc-serialize", "rand"]
+default = ["rational", "bigint"]
```

Comment:
Can you modify `.travis.yml` to test with features disabled and with only one of each individual feature enabled?